### DISCUSSION TITLE
Cannot add placeholder to the Primeng Listbox Filter

### DISCUSSION THREAD

**Discussion by pkoyande** [primefaces/community#1430] [PrimeNG] [answered]
I am using listbox of Primeng with the filter Attribute set to true. I have searched a lot but not able to set placeholder text to the built in filter. Cannot set through CSS also since it is an Input. Used [filterPlaceholder] = "some Text".. But it gives error as filterPlaceholder is not a known property. Please help me.. How do i add placeholder. I don't even see the search icon on the right side as shown in the documentation.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

The example below works, you can use it as `filterPlaceHolder`.
```
<p-listbox [options]="cities" [(ngModel)]="selectedCity" optionLabel="name" [filter]="true" filterPlaceHolder="hello world" [style]="{ width: '15rem' }" [listStyle]="{ 'max-height': '220px' }"></p-listbox>
```
